react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,39 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
-class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
-
-  render() {
-    const status = 'Next player: X';
-    return (
-      <div>
-        <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(3)}
-          {this.renderSquare(4)}
-          {this.renderSquare(5)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(6)}
-          {this.renderSquare(7)}
-          {this.renderSquare(8)}
-        </div>
+function Board ({ status, squares, handleClick }) {
+  const { boardRow } = styles;
+  return (
+    <Fragment>
+      <div className={styles.status}>{status}</div>
+      <div className={boardRow}>
+        <Square value={squares[0]} index={0} onClick={handleClick} />
```

Comment:
You are repeating this component call three times per row. Maybe you would do some array rendering, in order to avoid calling the same component many times.